src/module/protal/views/panduan-view.tsx

```
@@ -0,0 +1,94 @@
+            <Table className="min-w-full text-xs md:text-sm text-gray-700">
+              <TableHeader>
+                <TableRow className="bg-gradient-to-r from-blue-100 to-orange-100">
+                  <TableHead className="font-bold text-gray-800 uppercase tracking-wider px-2 py-2 w-2/3">Judul</TableHead>
+                  <TableHead className="font-bold text-gray-800 uppercase tracking-wider text-center px-2 py-2 w-1/3">Ukuran File</TableHead>
+                </TableRow>
+              </TableHeader>
+              <TableBody>
+                {panduanList.map((panduan: any) => (
+                  <TableRow key={panduan.id} className="even:bg-gray-50 odd:bg-white hover:bg-orange-50 transition">
+                    <TableCell className="font-semibold px-2 py-2 break-words max-w-xs">
+                      {panduan.file && panduan.file.url ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)